File: output/StarCoder/codenet/C++/Java/s472290172.java
import java.util.*;

public class s472290172 {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String str = sc.next();

        int max = 0, cnt = 0;
        for (char sc : str.toCharArray()) {
            if (sc == 'A' || sc == 'C' || sc == 'G' || sc == 'T') {
                cnt++;
            } else {
                cnt = 0;
            }
            max = max < cnt? cnt : max;
        }

        System.out.println(max);
    }
}

File: output/StarCoder/codenet/C++/Python/s472290172.py
str = raw_input()
max = 0
cnt = 0
for sc in str:
    if sc == 'A' or sc == 'C' or sc == 'G' or sc == 'T':
        cnt += 1
    else:
        cnt = 0
    max = max < cnt? cnt : max
print max